core/__tests__/models/property/property.ts

```diff
@@ -164,7 +164,7 @@ describe("models/property", () => {
     });
 
     test("keys cannot be from the reserved list of keys", async () => {
-      const reservedKeys = ["grouparooId", "grouparooCreatedAt"];
+      const reservedKeys = ["grouparooId", "grouparooCreatedAt", "_meta"];
```

Comment:
I can't quite figure out what `_meta` is for. Is this new?

Reply:
That's been there for a long time - check out https://github.com/grouparoo/grouparoo/blob/21ce6fc83cca60911b87b6d5d4824be43180e959/core/src/tasks/record/export.ts#L67-L80.  We use this to pass atypical information though an Import back out to the next time it's exported.  For example, if a Record is no longer supposed to be sent to a Destination (maybe it was removed from a Group), `import.data._meta.destinationId` would tell us that there should be one more export made to that destination